### DISCUSSION TITLE
HR Add-on?

### DISCUSSION THREAD

**Discussion by ravarkon** [YetiForceCompany/YetiForceCRM#16361] [Feature improvement]
I did not see noting about HR / as well time clock in and out, as well leave, sick days etc?
will this be added soon?

**Comment by apcloic** [comment]
@hzm170 
Check [Time Control](https://gitstable.yetiforce.com/index.php?module=OSSTimeControl&view=Calendar&mid=78&parent=76) module

**Reply by ravarkon** [reply]
What about HR where the staff can request leave ? Such as sick time and vacation time ?
